.github/workflows/main-integration-checks.yml

```diff
@@ -56,7 +56,7 @@ jobs:
           java-version: 1.11
 
       - name: Detekt
-        run: ./gradlew detekt
+        run: ./gradlew detektMain
```

Comment:
Detekt was broken due to the latest version changes introduced by the framework.